src/cli/commands/start.ts

```diff
@@ -212,8 +213,22 @@ export async function start(startContext: string | undefined, options: SWACLICon
   }
 
   if (options.ssl) {
-    if (options.sslCert === undefined || options.sslKey === undefined) {
-      logger.error(`SSL Key and SSL Cert are required when using HTTPS`, true);
+    if (options.sslCert === undefined && options.sslKey === undefined) {
+      logger.warn(`WARNING: Using built-in UNSIGNED certificate. DO NOT USE IN PRODUCTION!`);
```

Comment:
nit: not sure the production mention is needed here, as it's for `swa start` which is just for local emulation ;)